schemas/specimen.json

```diff
@@ -236,39 +243,52 @@
       "description": "Indicate a value, in decimals, that represents the percentage of infiltration by tumour cells in a specimen.",
       "valueType": "number",
       "meta": {
-        "core": true
+        "core": true,
+        "dependsOn": "sample_registration.tumour_normal_designation",
+        "note": "This field is only required if the specimen is tumour."
       }
     },
     {
       "name": "percent_proliferating_cells",
       "description": "Indicate a value, in decimals, that represents the count of proliferating cells determined during pathologic review of the specimen.",
       "valueType": "number",
       "meta": {
-        "core": true
+        "core": true,
+        "dependsOn": "sample_registration.tumour_normal_designation",
+        "note": "This field is only required if the specimen is tumour."
       }
     },
     {
       "name": "percent_inflammatory_tissue",
       "description": "Indicate a value, in decimals, that represents local response to cellular injury, marked by capillary dilatation, edema and leukocyte infiltration; clinically, inflammation is manifest by redness, heat, pain, swelling and loss of function, with the need to heal damaged tissue.",
       "valueType": "number",
       "meta": {
-        "core": true
+        "core": true,
+        "dependsOn": "sample_registration.tumour_normal_designation",
+        "note": "This field is only required if the specimen is tumour."
+      },
+      "restrictions": {
+        "required": true
```

Comment:
If this is only required when tumor, then required :true should be removed here,, like the other fields.

Reply:
I removed the "required:true" where there is a dependency.